### DISCUSSION TITLE
Why not stable websocket package in mobile app?

### DISCUSSION THREAD

**Discussion by xpwmaosldk** [immich-app/immich#225] [Q&A] [answered]
Hi there!

Why do you use unstable version(^2.0.0-beta.4-nullsafety.0) socket_io_client for websocket?

How about this?

https://docs.flutter.dev/cookbook/networking/web-sockets
and
https://pub.dev/packages/web_socket_channel

**Comment by alextran1502** [comment, marked as answer]
I haven't touched the WebSocket aspect for a while, I remember it was the compatibility issue for the combination of `websocket` and `redis` on the backend that only works with that version.

**Reply by xpwmaosldk** [reply]
Okay I will test for that (some day)
thanks for answer! :)
